styles/home.css

```
@@ -0,0 +1,164 @@
+.banner {
+  background-color: #cfe5ff;
+  height: 60vh;
+  text-align: center;
+  background-repeat: no-repeat;
+  background-position: bottom;
+  background-size: 130%;
```

💊 제안
background 이미지의 사이즈를 반응형단위로 가지고 가니 디자인과 다르게 나오는 것 같아요!
모바일에서는 contain 값으로 해당 부모만큼 늘어나도록 하고 PC 에서는 디자인에서의 이미지 사이즈를 주어
너무 크게 이미지가 늘어나지 않도록 해주시는 것을 추천드려요!

```suggestion
  background-size: contain;
```